.github/workflows/test.yaml

```diff
@@ -66,3 +66,41 @@ jobs:
             -e VERBOSE=-v \
             test-${{ matrix.python-version }} \
             ./test.sh
+
+  e2e:
+    name: e2e
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - "3.9"
+          - "3.12"
+        test-script:
+          - e2e/test_rebuild_following_bootstrap.sh
```

Comment:
Did you put the test-script in the test matrix because you plan to add additional test-scripts in future?

Reply:
Right, I'm anticipating scenarios like detecting a missing dependency and reporting the requirement that leads to it.